testdata/zk/Bank.sol

```diff
@@ -1,5 +1,5 @@
 // SPDX-License-Identifier: UNLICENSED
-pragma solidity ^0.8.18;
+pragma solidity 0.8.28;
```

Comment:
why a fixed version of solidity?

Reply:
if its to have a fixed bytecode hash I'd prefer we fetch it from the artifacts...

Reply:
It's for deterministic bytecode, but retrieving it isn't the only thing as the create2 addresses change on different bytecode, so we'd need to recompute them as well. Simpler to freeze the solc version for a simpler test case.